Tests/SwiftDocCTests/Semantics/Reference/RowTests.swift

```diff
@@ -29,8 +29,9 @@ class RowTests: XCTestCase {
             ["1: warning – org.swift.docc.HasAtLeastOne<Row, Column>"]
         )
         
+        XCTAssertEqual(renderBlockContent.count, 1)
         XCTAssertEqual(
-            renderBlockContent,
+            renderBlockContent.first,
```

Comment:
Should this not be in a separate patch?

Reply:
Split into a separate commit here: e4a6fc4041d7a6a1d18733aa3d3b564da7fd58c3.